src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -3218,12 +3247,78 @@ public ConstructorEdit(INamedTypeSymbol oldType)
             }
         }
 
+        private static bool TryAddParameterInsertOrDeleteEdits(ArrayBuilder<SemanticEditInfo> semanticEdits, ISymbol containingSymbol, SemanticModel? otherModel, EditAndContinueCapabilitiesGrantor capabilities, Func<SyntaxNode, SyntaxNode?>? syntaxMap, EditScript<SyntaxNode> editScript, CancellationToken cancellationToken, out bool notSupportedByRuntime)
```

Comment:
>containingSymbol, SemanticModel? otherModel [](http://example.com/codeflow?start=117&length=43)

I'd add doc comments on containingSymbol and otherModel parameters documenting that they are from old and new compilations, or vice versa.